### ISSUE TITLE
Build warnings in SDK 5.1.0 with Xcode 12 beta 6

### ISSUE THREAD

**Issue by idrougge** [launchdarkly/ios-client#220]
**Describe the bug**
`DiagnosticEvent.swift` has four warnings saying:
```
Immutable property will not be decoded because it is declared with an initial value which cannot be overwritten
```

**SDK version**
5.1.0

**Language version, developer tools**
Xcode 12, beta 6

**Comment by torchhound** [comment]
Hi @idrougge, thanks for reporting this bug.

_Tracked internally as 88282._

**Comment by gwhelanLD** [comment]
Fixed in the [5.2.0 release](https://github.com/launchdarkly/ios-client-sdk/releases/tag/5.2.0). Thanks again for reporting!
